rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/80_dense_vector_indexed_by_default.yml

```diff
@@ -0,0 +1,133 @@
+setup:
+  - skip:
+      version: ' - 8.9.99'
+      reason: 'dense_vector indexed by default was added in 8.10'
+
+---
+"Indexed by default with cosine similarity":
```

Comment:
Added YAML tests - some of them are already tested in unit tests, LMKWYT about removing incorrect params tests